src/webextension/survey/components/fields/sentiment.jsx

```diff
@@ -1,21 +1,58 @@
 import React, { Component } from 'react';
 
+export class StarRating extends Component {}
+
 export default class SentimentField extends Component {
+  handleChange(evt) {
+    this.props.input.onChange(evt);
+  }
+
+  renderInput(n) {
+    const { input } = this.props;
+    return (
+      <input
+        name={input.name}
+        type="radio"
+        id={`sentiment-${n}`}
+        value={n}
+        checked={input.value === n}
+        onChange={input.onChange}
+      />
+    );
+  }
+
+  renderLabel(n) {
+    return <label htmlFor={`sentiment-${n}`}>{n}</label>;
+  }
+
+  // This is a bit ugly; it's done this way to ensure that each <label> and
+  // <input> element are siblings, which lets us manage all of the star logic
+  // with CSS alone.
```

Comment:
+1, this is a bit ~~ugly~~ repetitive. Any way we can use a `for` loop or something?

```js
const stars = [];
for (let id = 5; id >= 1; id--) {
  stars.push(this.renderInput(id));
  stars.push(this.renderLabel(id));
}
```